### ISSUE TITLE
How to encrypt one single file?

### ISSUE THREAD

**Issue by gdhunt** [google/fscryptctl#45]
Fscrypt is a great encryption tool integrated directly into filesystem, and support transparent encryption of files and directories.

As far as I know, fscrypt works on empty directories.  Is there a method to encrypt one single file with fscrypt, and not create a empty directory?

**Comment by josephlr** [comment]
No, this is a restriction from the kernel, so there's nothing this user-space tool can do. When setting up a new policy, it has to be applied to an empty directory.
